apis/metal3.io/v1alpha1/doc.go

```diff
@@ -13,23 +13,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// Package v1alpha1 contains API Schema definitions for the metal3io v1alpha1 API group
-// +kubebuilder:object:generate=true
+// Package v1alpha1 contains API Schema definitions for the metal3.io/v1alpha1 API group
```

Comment:
This is not part of the metal3 project, so I don't think we should be appropriating that namespace.

Reply:
? we already have. This is just a comment spello